static/data/community.ts

```diff
@@ -15,6 +15,16 @@ const communityChat = {
   subtitle:
     'The Podman developers are generally around during CEST and Eastern Time business hours, so please be patient if you’re in another time zone!',
   links: [
+    {
+      text: 'Slack - Podman Community',
+      path: 'https://app.slack.com/huddle/T08PSQ7BQ/C08MXJLCFCN',
+      icon: 'logos:slack-icon',
+    },
+    {
+      text: 'Slack - Podman Developers',
+      path: 'https://app.slack.com/huddle/T08PSQ7BQ/C08NTKCDC1W',
```

Comment:
Crud, sorry, just spotted these two here.
```suggestion
      path: 'https://app.slack.com/client/T08PSQ7BQ/C08NTKCDC1W',
```